content/ecosystem/data.toml

```diff
@@ -1323,6 +1323,11 @@ name = "sdl2"
 source = "crates"
 categories = ["2drendering", "audio", "engines", "textrendering", "input", "windowing"]
 
+[[items]]
+name = "sdl3"
+source = "crates"
+categories = ["2drendering", "audio", "engines", "textrendering", "input", "windowing"]
```

Comment:
```suggestion
categories = ["2drendering", "audio", "engines", "3drendering", "input", "windowing"]
```

Reply:
This seems like a reasonable change - SDL3 by itself can't do any text stuff, but it can do 3D (via the new GPU API).